pyiron/sphinx/base.py

```diff
@@ -637,7 +637,7 @@ def _set_kpoints(
                 raise ValueError("n_trace has to be defined")
             high_symmetry_points = self.structure.get_high_symmetry_points()
             if high_symmetry_points is None:
-                raise ValueError("high_symmetry_points has to be defined")
+                raise ValueError("no 'high_symmetry_points' defined for 'structure'.")
```

Comment:
but does `self.structure.get_high_symmetry_points()` only returns `self.structure.high_symmetry_points`? Isn't the function then redundant ?

Reply:
I will rewrite the `high_symmetry_points` part in near future. For me it is fine at the moment.